sdks/python/apache_beam/io/lyft/s3_and_kinesis.py

```diff
@@ -0,0 +1,45 @@
+import json
+import logging
+
+from apache_beam import PTransform
+from apache_beam.pvalue import PBegin
+from apache_beam.pvalue import PCollection
+from apache_beam.transforms.core import Windowing
+from apache_beam.transforms.window import GlobalWindows
+
+
+class S3AndKinesisInput(PTransform):
+    """Custom composite transform that uses Kinesis and S3 as
+    input sources. This wraps the streamingplatform-dryft-sdk SourceConnector.
+    Only works with the portable Flink runner.
+    """
+
+    def expand(self, pbegin):
```

Comment:
I am not a 100% sure of this here. My understanding is that this would give us the PCollection which we could then pass to the DryftSDK.